common/response.js

```diff
@@ -14,14 +14,8 @@ function renderErrorView (req, res, msg = ERROR_MESSAGE, status = 500) {
   res.status(status)
   res.render('common/templates/error', {'message': msg})
 }
-function renderPaymentCompletedSummary (req, res, params) {
```

Comment:
Why was this removed?

Reply:
It is not used in the project

Reply:
Fair enough. Ideally this would have in a separate PR but I know how it is.